python/ray/_private/worker.py

```diff
@@ -129,6 +133,30 @@ def bind(
     ) -> "DAGNode[R]":
         ...
 
+    def options(
+        self: TSelfRemoteFunction,
```

Comment:
Is there a way to avoid the duplications and make it sync with the actual options supported in `remote_function.py`?

Reply:
Not currently, sadly. 😔 

At some point this might help:

> PEP 692 – Using TypedDict for more precise **kwargs typing

https://peps.python.org/pep-0692/

But that's still in draft. For now, there's no way to do it that I know of.